.github/dependabot.yml

```diff
@@ -55,6 +55,24 @@ updates:
       day: "sunday"
     assignees:
       - "Bento007"
+    groups:
+      database:
```

Comment:
FYI: this list is ordered from top to bottom. Dependabot will open a PR against the first group that matches.